web/customize_viewer.css

```diff
@@ -55,3 +184,28 @@
     margin-left: 72px;
   }
 }
+
+/**
+ * Implement our custom icons. Disable all of the default icons, since the
+ * buttons now have inline svg in viewer.html.
+ */
+.toolbarButton::before {
+  content: '' !important;
+  display: none !important;
+}
+
+i.svg-icon {
+  margin: 0;
+  overflow: auto;
+}
+
+i.svg-icon path,
+i.svg-icon rect,
+i.svg-icon circle,
+i.svg-icon polygon,
+i.svg-icon ellipse,
+i.svg-icon polyline,
+i.svg-icon line,
+i.svg-icon g {
+  fill: currentColor;
```

Comment:
Yeah, this is the same as `i.svg-icon *`. I think we're doing this so that the rendering engine can match css rules more quickly, though I'm not sure if that's a valid concern.